webapp-django/crashstats/crashstats/static/crashstats/css/tree.css

```diff
@@ -0,0 +1,61 @@
+/* From http://cssdeck.com/labs/pure-css-tree-menu-framework */
```

Comment:
I don't think you can legally lift that. There's no indication that you're allowed to copy it on that page or on that site as far as I can tell. Under what license is that material distributed under?